### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-matrix/) for Kodi add-ons: look and feel/visualization.matrix.

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by gade01 — APPROVED** [review]

File: visualization.matrix/resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -5,129 +5,130 @@
...
 "Content-Transfer-Encoding: 8bit\n"
-"Language: de_DE\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.7.1\n"
 
 msgctxt "#30000"
 msgid "Allgemein"
-msgstr ""
+msgstr "Allgemein"
```
Comment (ksooo) [inline visualization.matrix/resources/language/resource.language.de_de/strings.po:22]: @gade01 This looks all wrong! All the source strings are de_de, not en_gb!

Comment (gade01) [reply visualization.matrix/resources/language/resource.language.de_de/strings.po:22]: @ksooo Yup, it's like that in the de_de file itself.
Not an issue with Weblate: https://github.com/xbmc/visualization.matrix/blob/Matrix/visualization.matrix/resources/language/resource.language.de_de/strings.po

Comment (gade01) [reply visualization.matrix/resources/language/resource.language.de_de/strings.po:22]: https://github.com/xbmc/visualization.matrix/commit/2864cc594d6b713bebbe1f7efee2694f2794dd27#diff-2d69d1e4db466a1339c742e06609a331fdba93687388bb390667b439d052cf8d

Comment (ksooo) [reply visualization.matrix/resources/language/resource.language.de_de/strings.po:22]: Any idea how to fix this?

Comment (gade01) [reply visualization.matrix/resources/language/resource.language.de_de/strings.po:22]: Hmm.. either delete the file in a PR and have Weblate recreate it.
After that translation memory can be used to translate all strings again.

Or do another PR to manually replace de_de with en_gb in the de_de file (that will probably remove translations at Weblate).

**Pull request merged**